circle.yml

```diff
@@ -24,8 +28,50 @@ jobs:
       - run:
           name: Run integration tests
           command: make test-integration
+  lint_terraform:
+    docker:
+      - image: hashicorp/terraform:light
+    steps:
+      - checkout
+      - run: terraform fmt -check=true fastly/terraform/
+  deploy_fastly_service_production:
+    docker:
+      - image: hashicorp/terraform:light
+    steps:
+      - checkout
+      - run: terraform init fastly/terraform/
+      - run: terraform import -config=fastly/terraform fastly_service_v1.app 7mnWDqaHxkKwIFASbvnV13
+      - run: terraform apply --auto-approve fastly/terraform/
 workflows:
   version: 2
   test:
     jobs:
-      - test
+      - test:
+          filters:
+            tags:
+              ignore: /^v.*/
+            branches:
+              ignore: master
+      - lint_terraform:
+          filters:
+            tags:
+              ignore: /^v.*/
+            branches:
+              ignore: master
+      - deploy_fastly_service_production:
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
We don't tag `v.` releases for registry ui. THe Github user `origami-auto-release` adds a tag, but I'm actually not sure how that happens

Reply:
We will in a subsequent pull-request :-)